pkg/scheduler/scheduler.go

```diff
@@ -388,7 +428,7 @@ func (k *KubernetesScheduler) StatusUpdate(driver bindings.SchedulerDriver, task
 		}
 		k.RLock()
```

Comment:
can safely remove this RLock/RUnlock since the only thing they were guarding is now protected via `slaveStorage`.